app/models/organisation.rb

```diff
@@ -19,4 +20,8 @@ def self.fetch_organisations_from_register
       organisations_gateway: Gateways::GovukOrganisationsRegisterGateway.new
     ).execute.sort
   end
+
+  def team_list
```

Comment:
Would you mind walking me through this? This data model looks a bit windy to manage the user x organisation relation